taca/deliver/cli.py

```diff
@@ -0,0 +1,100 @@
+""" CLI for the deliver subcommand
+"""
+import click
+from ngi_pipeline.utils.communication import mail_analysis
+from taca.deliver import deliver as _deliver
+
+@click.group()
+@click.pass_context
+@click.option('--deliverypath', type=click.STRING,
```

Comment:
I agree. However, I made a quite hacky thing with having arbitrary placeholders in the configuration, which will be expanded using instance attributes. That makes it hard to use hyphens. I'm not too happy with my implementation though, not very pythonic.. I'll probably modify that.